src/spinegeneric/cli/check_data_consistency.py

```diff
@@ -111,6 +115,7 @@ def main():
         ]
     )
 
+    tsv_file = pd.read_csv(str(args.path_in / "participants.tsv"), sep="\t")
```

Comment:
Note to self: We still use `pandas` for `schema.validate`, double-check if this gets removed later on.